src/status_im/ui/screens/communities/community_overview_redesign.cljs

```diff
@@ -68,17 +70,41 @@
                :render-fn                         channel/list-item}]]])
          list-of-channels)]])
 
+(defn icon-color []
+  (colors/theme-colors
+   colors/white-opa-40
+   colors/neutral-80-opa-40))
+
 (defn community-card-page-view [{:keys [name description locked joined
-                                        status tokens cover tags] :as community}]
+                                        status tokens cover tags community-color] :as community}]
   [react/view
    {:style
     {:flex 1
      :border-radius   20}}
    [react/view (styles/community-cover-container 148)
+
     [react/image
      {:source      cover
-      :style  {:position :relative
-               :flex 1}}]]
+      :style  {:position :absolute
+               :flex 1}}]
+    [react/view {:style {:margin-top 26}}
```

Comment:
`rn/view`

There are a few `react/view` in the updated files.

Reply:
will adjust :)